packages/app/src/cli/utilities/app/http-reverse-proxy.ts

```diff
@@ -91,10 +100,29 @@ ${output.token.json(JSON.stringify(rules))}
   abortController.signal.addEventListener('abort', () => {
     server.close()
   })
+
   await Promise.all([
     renderConcurrent({
       processes: [...processes, ...additionalProcesses],
       abortController,
+      onInput(input: string) {
+        if (input === 'p' && previewUrl) {
+          // eslint-disable-next-line @typescript-eslint/no-floating-promises
+          openURL(previewUrl)
+        } else if (input === 'q') {
+          process.exit(0)
+        }
```

Comment:
Is this another way of opening the URL? we are not showing these options to the user no?

Reply:
There are no other ways it's only hotkeys. The user will have to press a key to continue and open the URL so there is no chance they miss this. If they close the window then they can reopen it by pressing P.

Reply:
I've added a preview link at the bottom of the logs.

Reply:
Sorry, i mean other way because we also have the `enter` one right?